### ISSUE TITLE
Review nginx and mysql configurations to ensure optimal performance

### ISSUE THREAD

**Issue by tannerjfco** [drud/ddev#239]
## What happened (or feature request):
Per #195 We should review our nginx and mysql configurations to ensure configurations that relate to or impact performance are configured in the most optimal way possible. 

## What you expected to happen:
When we upgraded the web containers to PHP7, time was spent to focus on providing configurations for opcache, memory limits, etc to ensure we provided optimal performance. The same should be done for nginx and mysql, reviewing if there are any configurations that can be adjusted or introduced that would improve general web application performance. We should also check to see if new releases improve performance at all, and if so upgrade to them.

## Related source links or issues:

**Comment by rickmanelius** [comment]
Thanks for moving on this! A couple ideas:

* Is it worth documenting any refence material that justifies any recommendations? That would help in the review process as well as ongoing performance audits in the future. Example: there are many Apache tuning articles on how to change number of threads based on RAM pool and php.ini limits.
* Not sure if it's worth expanding this to address docker container resource allocations.

**Comment by rfay** [comment]
The fairly standard nginx config for Drupal is [Perusio's](https://github.com/perusio/drupal-with-nginx) - I think it's been holding as "normative" for many years now. Not sure if it's worth fiddling with using it, especially since we span more than one distro.

**Comment by tannerjfco** [comment]
I originally created this ticket as part of an initial effort to break out the tasks in #195. However, when we discussed the tasks in sprint planning, @beeradb suggested that this task wasn't worth pursuing at this time. Since we know our biggest performance issues relate to bind mount performance on Mac, it seems questionable that we would make any notable gains in terms of nginx or mysql configuration changes. I also don't have any specific configuration changes in mind at this time to evaluate in terms of performance. Given this, I've marked this incubate and needs decision. We should decide if this task is worth pursuing (possibly at a later time), and if so, research/review would be useful to determine if there are any configurations worth evaluating.
